### ISSUE TITLE
VeraCrypt System Encryption Pretest Failed (a new one) 

### ISSUE THREAD

**Issue by AARRGGHHH** [veracrypt/VeraCrypt#196]
This is occurring on a PC with two Windows installations on separate drives.  The critical one with work files is encrypted.  I would like to encrypt the other drive (and keep it bootable).  

When running "System --> Encrypt System Partition/Drive...", the VeraCrypt Volume Creation Wizard reports "The VeraCrypt system encryption pretest failed...".  I only get a password prompt for my previously encrypted drive, I never get a prompt for the other drive.  

The error message implies that the OS does not boot from the drive on which the OS is installed.  This is not the case.  The OS was booting from this drive for a long time before the encrypted drive was even purchased, and I have made no changes since then as to how the other drive boots.  

For the sake of discussion, I tried clicking both yes and then no to the Boot Drive options in the System Volume setup in VeraCrypt, this made no difference, it returned the same error message.  

Long story short, I never get a password prompt on reboot, and the encryption never runs. VC 1.19 64 bit. Win 7 64 bit

Thank you
